README.md

```diff
@@ -28,20 +28,22 @@ specification to describe this ability as well, and move the document to the sta
   1. Updating state:
      This is the data required for the correct functioning of the app. Examples:
      * Updated search index for a search app.
-     * Pushing a critical application update.
-     * Updated icons after a UI overhaul of a web app.
+     * A critical application update.
+     * Updated icons.
 
   2. Updating content:
      Periodic content producers can push content to user devices, to be consumed at a later, more
      convenient time. Examples:
-     * Fresh articles from news sites.
+     * Fresh articles from news sites. Push API can be used here, but the updates will be too
+     frequent, so the content site will have to batch the push messages. The browser is in a much
+     better position to do that.
      * List of badges the user has earned, in a fitness app.
      * List of new songs from their favorited artists, in a songs app, to be downloaded using
-        Background Fetch if the user clicks on "Download".
+     Background Fetch if the user clicks on "Download".
 
   Currently, the Push API enables this, but it requires setting up a server that speaks the Web
-  Push protocol. Additional tasks like resource management add complexity to this approach. Periodic
-  Background Sync offers a simpler, more accessible solution.
+  Push protocol, and managing resources to keep the server available. Periodic Background Sync
```

Comment:
Does "and managing resources to keep the server available" mean "maintaining server logic"?

I think I keep getting hung up on "resources". Does it mean, CPU, memory, disk space… but I think it means you'd have to maintain a backend architecture rather than just having a static site.

Reply:
Yeah, maintaining backend architecture is what I mean. If that's too obvious I'll remove this sentence.

Reply:
I think it's worth mentioning, but I'd just say "maintaining a backend"

Reply:
Done, thanks!